### ISSUE TITLE
Add list of Submitted Pubs to Submissions tab

### ISSUE THREAD

**Issue by idreyn** [pubpub/pubpub#1615]
Depends on #1620 to load the right Pubs

- Steal Overview tab look and feel
- Reuse/generalize `OverviewSearchGroup` and `PubRow`
- Add new prop to `OverviewSearchGroup` to allow us to specify a custom set of tabs (using the `Filter[]` type in that component's file)

Out of scope:
- Adding submission-specific detail elements or controls to each Pub
- Sorting the Pubs by Submission timestamp
